src/Evolution/Systems/ScalarWave/Equations.hpp

```diff
@@ -103,33 +103,38 @@ struct ComputeNormalDotFluxes {
  * \brief Computes the penalty flux
  *
  * The penalty flux is given by:
+ *
  * \f{align}
  * F^*(\Psi) =& F(\Psi) = 0 \\
- * F^*(\Pi) =& F(\Pi)_{\mathrm{int}} + \\
- *         \frac{1}{2} p \left(U^-_{\mathrm{ext}} - U^-_{\mathrm{int}}\right) \\
- * F^*(\Phi_i) =& F(\Phi_i)_{\mathrm{int}} - \\
- *         \frac{1}{2} p \left(U^-_{\mathrm{ext}} - U^-_{\mathrm{int}}\right)
- * \f}
- * where \f$F^*\f$ is the normal dotted with the numerical flux and \f$F\f$ is
- * the normal dotted with the flux, which is computed in
- * ScalarWave::ComputeNormalDotFluxes.
+ * F^*(\Pi) =& F(\Pi)_{\mathrm{int}} +
+ *         \frac{1}{2} p \left(v^{\hat -}_{\mathrm{ext}} - v^{\hat
+ * -}_{\mathrm{int}}\right) \\ F^*(\Phi_i) =& F(\Phi_i)_{\mathrm{int}} -
+ *         \frac{1}{2} p \left(v^{\hat -}_{\mathrm{ext}} - v^{\hat
+ * -}_{\mathrm{int}}\right) \f}
+ *
+ * where \f$F^*\f$ is the interface normal dotted with numerical flux, \f$F\f$
+ * is the normal dotted with flux (computed in
+ * ScalarWave::ComputeNormalDotFluxes), and \f$v^{\hat -}\f$ is an incoming
```

Comment:
Sorry, I'm confused about the `hat` over the `-`. I expected just `v^{-}_{\mathrm{ext}}`. Could you explain that to me on GitHub here?

Reply:
I have kept the `hat` everywhere, both in the ScalarWave and CurvedScalarWave systems. See e.g. their Characteristics. This started from the consensus posted [here](https://github.com/sxs-collaboration/spectre/pull/1751#discussion_r349254666) by Larry

Reply:
That's not what was agreed on as far as I understand. What I thought we were agreeing to was `v_{\hat{\alpha}}` is used as a place holder for ALL char fields. Individual char fields would be, e.g. `v^-`, `v^+`, `v^\Psi` (without the hat). @kidder did you and I not communicate this clearly with each other on this?

Reply:
My comment was intended only for denoting all fields; I don't see a reason to include the hat on a specific field.

Reply:
:hear_no_evil:  Removed hats for individual fields.